src/notation/internal/notationuiactions.cpp

```diff
@@ -213,7 +214,7 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-visible",
              mu::context::UiCtxNotationOpened,
              QT_TRANSLATE_NOOP("action", "Toggle visibility of elements"),
-             QT_TRANSLATE_NOOP("action", "Toggle visibility of elements")
+             QT_TRANSLATE_NOOP("action", "Toggle visibility of elements"),
```

Comment:
That comma is not needed. Actually the whole line is not needed (and then the comma on the line abobe is not needed anomre either)